packages/config/src/VitessceAutoConfig.test.js

```diff
@@ -511,22 +526,468 @@ describe('src/VitessceAutoConfig.js', () => {
           },
           x: 0,
           y: 0,
-          w: 12,
-          h: 12,
+          w: 6,
+          h: 4,
+        },
+        {
+          component: 'obsSetSizes',
+          coordinationScopes: {
+            dataset: 'A',
+          },
+          h: 4,
+          w: 6,
+          x: 6,
+          y: 0,
+        },
+        {
+          component: 'obsSetFeatureValueDistribution',
+          coordinationScopes: {
+            dataset: 'A',
+          },
+          h: 4,
+          w: 6,
+          x: 0,
+          y: 4,
+        },
+        {
+          component: 'heatmap',
+          coordinationScopes: {
+            dataset: 'A',
+          },
+          h: 4,
+          props: {
+            transpose: true,
+          },
+          w: 6,
+          x: 6,
+          y: 4,
+        },
+        {
+          component: 'featureList',
+          coordinationScopes: {
+            dataset: 'A',
+          },
+          h: 4,
+          w: 6,
+          x: 0,
+          y: 8,
         },
       ],
       initStrategy: 'auto',
     };
 
-    const config = await generateConfigs(urls);
+    const config = await generateConfig(urls, {});
     expect(config).toEqual(expectedConfig);
   });
 
   it('raises an error when URL with unsupported file format is passed', async () => {
     const urls = ['http://localhost:4204/@fixtures/zarr/anndata-0.7/somefile.zarr'];
 
-    await generateConfigs(urls).catch(
-      e => expect(e.message).toContain('This file type is not supported.'),
+    await generateConfig(urls, NO_HINTS_CONFIG, '', false).catch(
```

Comment:
Are there extra parameters here?

Reply:
No, there aren't. I forgot to refactor this code and remove the extra parameters. Thanks for catching it!